types/dom-clipboard-api/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for w3c Clipboard API 1.0
+// Project: https://w3c.github.io/clipboard-apis/
+// Definitions by: James Garbutt <https://github.com/43081j>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface Clipboard extends EventTarget {
```

Comment:
Oh I see what you're doing here. Am I correct in understanding that you want to define types for a future version of the w3c dom spec, before it would be appropriate for them to be added to the TypeScript lib?

Reply:
yes exactly. we don't put w3c drafts in the generator but rather place them here until they reach candidate recommendation. 

i haven't submitted a DOM API here before though so this may well be wrongly structured..